client/utils.ts

```diff
@@ -66,6 +67,10 @@ export function arrayChunked<T>(a: T[], chunkSize: number): T[][] {
   }, [] as T[][]);
 }
 
+export function capitalize(s: string): string {
+  return `${s.charAt(0).toUpperCase()}${s.slice(1)}`;
+}
```

Comment:
BTW, I've found [change-case](https://www.npmjs.com/package/change-case) to be a pretty handy toolbox for string utils. `sentenceCase` should work here.